__tests__/app.test.js

```diff
@@ -111,4 +112,84 @@ describe("app", () => {
 
     })
 
+    describe("POST /api/articles/:article_id/comments", () => {
```

Comment:
2 more tests we want to add here.
test for if the article ID is invalid - ie it is a string (400)
test for it the username is not valid (isn't in the users table) - should throw a PSQL error that can be dealt with - 404